arches/app/views/etl_manager.py

```diff
@@ -19,15 +19,73 @@ class ETLManagerView(View):
     to get the ETL modules from db
     """
 
+    def dictfetchall(self, cursor):
```

Comment:
We should probably try to avoid using Python types in function names. Maybe this could be: `fetch_load_errors(self, cursor)`